swap_meet/clothing.py

```diff
@@ -0,0 +1,17 @@
+from swap_meet.item import Item
+from swap_meet.vendor import Vendor
+
+
+class Clothing(Item):
+    """ Child class inherits condition descriptions from Items class"""
+    def __init__(self,category="",condition=None):
+        self.category="Clothing"
+        self.condition=condition
```

Comment:
this works! I think we can follow some more inheritance guidelines, though. `self.category` will always being "clothing," so we don't even need to let the user pass in that argument.

Plus, the parent class Item already has a `__init__` method we can use so we aren't repeating code:

```suggestion
    def __init__(self,condition=None):
        super().__init__("clothing",condition)
```